### PR TITLE
Feature/17 2 create component timer in tests view

### PR DESCRIPTION
**Pull request by visiteur85**

### CODE REVIEW

File: src/components/Timer/Timer.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        minutes: numberWithZero(minutes),
+      });
+    }
+  };
+  const startTimer = () => {
+    setTimer(!timer);
+  };
+  useEffect(() => {
+    if (timer) {
+      const intervalId = setInterval(() => {
```
Comment (macroserP) [inline src/components/Timer/Timer.tsx:43]: try to use setTimeout instead
https://learn.javascript.ru/settimeout-setinterval
but it is not crucial

File: src/components/Timer/Timer.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        return '0'.repeat(2 - numberString.length) + numberString;
+      };
+
+      setTime({
+        seconds: numberWithZero(seconds),
+        minutes: numberWithZero(minutes),
+      });
+    }
+  };
+  const startTimer = () => {
```
Comment (macroserP) [inline src/components/Timer/Timer.tsx:38]: try to implement pause timer functionality

File: src/components/Timer/Timer.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+
+import dayjs from 'dayjs';
+
+const timeDefault = {
```
Comment (macroserP) [inline src/components/Timer/Timer.tsx:5]: better to move it to storybook controls and add endDate as a props

File: src/components/Timer/Timer.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  minutes: '20',
+  seconds: '00',
+};
+
+export const Timer = () => {
+  const [time, setTime] = useState(timeDefault);
+  const [timer, setTimer] = useState(false);
+
+  let endDate = dayjs().add(19, 'minute');
```
Comment (macroserP) [inline src/components/Timer/Timer.tsx:14]: hardcoded values

**Review by otabek — APPROVED** [review]

**Review by Zenater — APPROVED** [review]

**Pull request merged**